__test__/itinerary.test.js

```diff
@@ -0,0 +1,20 @@
+const Itinerary = require('../src/itinerary')
+const Port = require('../src/port.js')
+
+describe('itinerary constructor', () => {
+    it('it returns an object', () => {
+        
+        expect(new Itinerary('Itinerary')).toBeInstanceOf(Object);
```

Comment:
remove empty line on line 6. Only have an empty line before the expect statement if you are doing some setup beforehand.